Cargo.toml

```diff
@@ -25,7 +25,7 @@ multipart = {version = "0.18.0", default-features = false, features = ["client"]
 native-tls = {version = "0.2.10", optional = true}
 rustls-native-certs = { version = "0.7", optional = true}
 rustls-opt-dep = {package = "rustls", version = "0.22.0", optional = true}
-rustls-pki-types = "1"
+rustls-pki-types = {version = "1", optional = true}
```

Comment:
I don't this dependency is necessary at all as the types are re-exported as `rustls::pki_types`.

Reply:
The re-exporting was introduced at `rustls` 0.22.1. So when we want to use `rustls-pki-types` crate via `rustls`, we have to set the `rustls` minimum version requirement as 0.22.1 instead of 0.22.0. Which option is appropriate for `attohttpc`?

Reply:
Since 0.22.1 is fully semver-compatible with 0.22.0, I see little downside to bumping `rustls`. Most leaf projects will regularly run `cargo update` anyway so that the final result would be the same.

Reply:
Sure. I will change to use `rustls-pki-types` via `rustls` crate.

Reply:
Updated.